recipes/libx265/all/conanfile.py

```diff
@@ -16,13 +17,15 @@ class Libx265Conan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "assembly": [True, False],
```

Comment:
I guess that `nasm` should be added in `build_requirements()` if `assembly=True`?
https://github.com/videolan/x265/blob/241342f25bd1a83678b24588712f91ca0bff99f3/source/CMakeLists.txt#L352

Reply:
Good catch, I didn't realize I had nasm installed on my system.
Fixed this and enabled `assembly` by default.